teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, _name: 'Unknown', _parent_permission: true)
+    super(age, _name: 'Unknown', _parent_permission: true)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
+
+teacher = Teacher.new('Math', 25)
+puts teacher.can_use_services?
```

Comment:
- [ ] [OPTIONAL] You added this piece of code to your codebase I assume to test your code, please make sure to clean up your code by removing unused pieces of code before sending it for review.